pypeit/flatfield.py

```diff
@@ -305,31 +409,8 @@ def run(self, armed=False):
             #                        arqa.plot_orderfits(self, msblaze, flat_ext1d, desc="Blaze function")
             artracewave.plot_orderfits(self.setup, msblaze, flat_ext1d, desc="Blaze function")
         #
-        '''
+   """
 
         # Return
         return self.mspixelflatnrm, self.slit_profiles
-
-    def show(self, attr='norm', display='ginga'):
-        """
-        Show one of the internal images
-
-        Parameters
-        ----------
-        attr : str
-          mspixelflat -- Show the combined flat image, unnormalized
-          norm -- Show the combined normalized flat image
-        display : str, optional
-
-        Returns
-        -------
-
-        """
-        if attr == 'mspixelflat':
-            if self.mspixelflat is not None:
-                ginga.show_image(self.mspixelflat)
-        elif attr == 'norm':
-            if self.mspixelflatnrm is not None:
-                ginga.show_image(self.mspixelflatnrm)
-
-
+'''
```

Comment:
Files should end with at least one empty line.

Reply:
Done.